guides/doc-Project_API/topics/con_oauth-authentication-overview.adoc

```diff
@@ -0,0 +1,15 @@
+[id="oauth-authentication-overview"]
+= OAuth authentication overview
+
+As an alternative to basic authentication, you can use limited OAuth 1.0 authentication.
+This is sometimes referred to as _1-legged OAuth_ in version 1.0a of the protocol.
+
+To view OAuth settings, in the {ProjectWebUI}, navigate to *Administer* > *Settings* > *Authentication*.
+The *OAuth consumer key* is the token to be used by all OAuth clients.
+
+{Project} stores OAuth settings in the `/etc/foreman/settings.yaml` file.
+Use the `{foreman-installer}` script to configure these settings, because {Project} overwrites any manual changes to this file when upgrading.
```

Comment:
```suggestion
Use the `{foreman-installer}` script to change these settings.
```

I'd drop the latter half of the sentence because we don't want to give the idea that users should manually edit this file.

Option is `--foreman-oauth-consumer-key`; maybe we can mention this here or in a related procedure.

Reply:
The following procedures tells the user how to list all OAuth installer options. I'm keeping the option out.